pkg/api/apiloader_test.go

```diff
@@ -487,23 +294,180 @@ func TestSerializeContainerService(t *testing.T) {
   },
   "type": "sampleType",
   "properties": {
-    "kubernetesVersion": "1.11.6",
-    "dnsPrefix": "blueorange",
-    "fqdn": "blueorange.westus2.azure.com",
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
```

Comment:
We can drop the `"orchestratorType": "Kubernetes"` line since it's the default after #4038 merged.

Reply:
We actually can't easily because this is the RAW, generated JSON output that we expect after putting data through a bunch of serialization/deserialization.

We can update the mock data so it doesn't specify the orchestratorType property, but then something in the serialization path renders a `"orchestratorType": ""` in the final JSON, which is (IMO) weirder.

(probably can ultimately be solved but I'm not gonna worry about it)

Reply:
Ah, right. Never mind!